static/css/timetable/partials/search_bar.scss

```diff
@@ -363,30 +421,38 @@
 }
 
 .trans50 {
-  // This makes it semi transparent
-  background-color: $white-transparent-0;
-  border: 1px solid $white-transparent-0;
-  border-top-color: transparent;
-  box-shadow: none;
-
+  @include theme() {
+    // This makes it semi transparent
+    background-color: $white-transparent-0;
+    border: 1px solid $white-transparent-0;
+    border-top-color: transparent;
+    box-shadow: none;
+  }
```

Comment:
Include theme here even though I don't use a variable because .trans50 updates the styles of other classes and I need the updated styles here to be in the same scope (if I don't include theme, e.g. background-color won't become $white-transparent-0)

Reply:
tldr: since `background-color` in [search-results](https://github.com/jhuopensource/semesterly/blob/2119b788012414812eecb3448b4bfc6332f7b242/static/css/timetable/partials/search_bar.scss#L150-L151) is defined inside `theme()`, I need to do the same here to update `background-color` in search-results